kernel/src/scan/state.rs

```diff
@@ -66,6 +66,20 @@ impl DvInfo {
             .transpose()?;
         Ok(dv_treemap.map(treemap_to_bools))
     }
+
```

Comment:
need doc comment. in particular, we should make clear that the indicies here are _to be removed_ from the result. (i think ;) )

Reply:
Also done